diff --git a/iam/principals.d.ts b/iam/principals.d.ts
index 48fa83c1a9145decc973543b5f393220daf1b005..601f969076089996d7680745a0cabb6ddb308726 100644
--- a/iam/principals.d.ts
+++ b/iam/principals.d.ts
@@ -1,5 +1,5 @@
 import { Principal } from "./documents";
-export declare module Principals {
+export declare namespace Principals {
     /**
      * Service Principal for Amazon Certificate Manager
      */
